assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidgetV2/UserDimensionsPieChart.js

```diff
@@ -162,13 +182,24 @@ export default function UserDimensionsPieChart( { dimensionName, entityURL, sour
 				</p>`
 			);
 
-			if ( sourceLink && rowData[ 0 ].toLowerCase() === 'others' ) {
-				tooltip += (
-					`<p>
-						<a class="googlesitekit-cta-link googlesitekit-cta-link--external googlesitekit-cta-link--inherit" href="${ sourceLink }" target="_blank" rel="noreferrer noopener">
-							${ __( 'See the detailed breakdown in Analytics', 'google-site-kit' ) }
-						</a>
-					</p>`
+			if ( sourceLink && rowLabel === 'others' ) {
```

Comment:
`others` need to be translated because we set it manually when parse chart data and use translation there:

```suggestion
			const othersLabel = __( 'Others', 'google-site-kit' ).toLowerCase();
			if ( sourceLink && rowLabel === othersLabel ) {
```

Reply:
👍